packages/jest-core/src/lib/activeFiltersMessage.ts

```diff
@@ -27,7 +27,7 @@ const activeFilters = (
 
     const messages = ['\n' + chalk.bold('Active Filters: ') + filters];
 
-    return messages.filter(message => !!message).join(delimiter);
+    return messages.filter(message => Boolean(message)).join(delimiter);
```

Comment:
you could have this shorter and receive`Boolean` wrapper as a reference
something like
`return messages.filter(Boolean).join(delimiter);`

Reply:
yes i tried to do that 😀

Reply:
I think a `return messages.filter(message => message.length > 0).join(delimiter);` is better